gateway/1.5/modules/ROOT/pages/flex-sizing-guide.adoc

```diff
@@ -0,0 +1,62 @@
+= Reviewing Resource Sizing 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:table-caption!:
+
+Before getting started with Flex Gateway, you must analyze and choose your configuration to obtain the best performance and auto-scaling rate.
+
+When assigning resources for a node, you must consider:
+
+* Number of APIs
+* Amount of memory the node needs
+* Processing power for the node
+* Number of policies in each API
+* Amount of traffic each node handles
+* Amount of time the node takes to answer the request (latency)
+* Type of traffic the node handles, and within the traffic:
+** The number of requests per second the node handles
+** Size of each request (bytes)
+
+== Choose Appropriate Resources
+The following table guides you to know the resources you must allocate to your node according to your environment:
```

Comment:
```suggestion
== Choose Appropriate Resources

The following table guides you to know the resources you must allocate to your node according to your environment:
```
I think you need this empty line for the Header to render properly. Please confirm in a local build.

Reply:
OK! Thanks!